c/driver/postgresql/postgresql_test.cc

```diff
@@ -1283,6 +1280,23 @@ class PostgresTypeTest : public ::testing::TestWithParam<TypeTestCase> {
 };
 
 TEST_P(PostgresTypeTest, SelectValue) {
+  std::string value = GetParam().sql_literal;
+  if ((value == "'-inf'") || (value == "'inf'")) {
+    const uint32_t info_code = ADBC_INFO_VENDOR_VERSION;
```

Comment:
I wonder if we should put a function in utils.c for `AdbcGetInfoVendorVersion`

Reply:
#598 is somewhat related

Reply:
It seems the check needs tweaking, though